test/src/org/labkey/test/tests/cds/CDSMAbGridTest.java

```diff
@@ -0,0 +1,190 @@
+package org.labkey.test.tests.cds;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.rules.Timeout;
+import org.labkey.test.Locator;
+import org.labkey.test.pages.cds.AntigenFilterPanel;
+import org.labkey.test.pages.cds.MAbDataGrid;
+import org.labkey.test.util.cds.CDSHelper;
+import org.openqa.selenium.WebElement;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+import static org.labkey.test.pages.cds.MAbDataGrid.*;
+
+@Category({})
+public class CDSMAbGridTest extends CDSReadOnlyTest
+{
+    private final CDSHelper cds = new CDSHelper(this);
+
+    @Before
+    public void preTest()
+    {
+        cds.enterApplication();
+        cds.ensureNoFilter();
+        cds.ensureNoSelection();
+
+        cds.goToAppHome();
+    }
+
+    @Override
+    public BrowserType bestBrowser()
+    {
+        return BrowserType.CHROME;
+    }
+
+    @Override
+    public List<String> getAssociatedModules()
+    {
+        return Arrays.asList("CDS");
+    }
+
+    @Override
+    public Timeout testTimeout()
+    {
+        return new Timeout(60, TimeUnit.MINUTES);
+    }
+
+    @Test
+    public void testMAbPage()
+    {
+        CDSHelper.NavigationLink.MABGRID.makeNavigationSelection(this);
+        MAbDataGrid grid = new MAbDataGrid(getGridEl(), this, this);
+
+        log("Verify subject based info pane presence for mAb and other tabs");
+        Locator.XPathLocator subjectInfoPane = CDSHelper.Locators.subjectInfoPaneHeader().notHidden();
+        assertElementNotPresent(subjectInfoPane);
+        CDSHelper.NavigationLink.GRID.makeNavigationSelection(this);
+        assertElementPresent(subjectInfoPane);
+        CDSHelper.NavigationLink.MABGRID.makeNavigationSelection(this);
+
+        log("Verify mAb summary grid content");
+        grid.clearAllFilters();
+        Assert.assertEquals("Number of mab/mabmix rows is not as expected", 54, grid.getMabCounts());
+        Assert.assertEquals("Geometric mean value for '2F5' is not as expected", "1.50583", grid.getMabCellValue("2F5", GEOMETRIC_MEAN_IC50_COL));
```

Comment:
Purely a style/subjective comment...
If I could go back and change one thing about test code I have written it would be to not assert (and possibly fail) on issues that shouldn't block checks that follow in the test.
If the number of rows is not as expected I agree we should fail, but I think it would also be useful to see if the MabCellValue of 2F5 is as expected as well. What I try to do in tests I am currently working on is setting a boolean value if an expected condition is not met, continue with the test, and then check the boolean at the end. 
I'm not saying you should change this code just something to keep in mind fo future tests.